docs: document trusted boot (#2612)
spectrocloud/librarium: docs/docs-content/clusters/edge/trusted-boot/key-management.md

are signed with the authentic private key and can be trusted.

The private PK must be stowed away in a secure location **immediately** after being generated. You do not need the
private key during EdgeForge, installation or deployment of your Edge devices. The public PK key is required during the
EdgeForge build process so that it can be embedded into the Edge Installer ISO.

## Key Exchange Key (KEK)

Similar to how PK authenticates updates to the KEK key, the KEK authenticates updates to the Signature Database (DB). It
also consists of pair that has a private key and a public key.

The private KEK signs updates to the DB. When there are updates to the DB, the public KEK is used to verify that those
updates are signed by the authentic private KEK.

The private KEK must be stowed away in a secure location **immediately** after being generated. You do not need the
private key during EdgeForge, installation or deployment of your Edge devices. The public KEK is required during the
EdgeForge build process so that it can be embedded into the Edge Installer ISO.

## Signature Database (DB) Key

> The signature database (DB) stores The db contains the authorized signatures or certificates for valid Unified Kernel

Reviewer: ```suggestion
The signature database (DB) stores The db contains the authorized signatures or certificates for valid EFI
```

> Image (UKI) files. The DB key is also a pair that has a public key and a private key.

Reviewer: ```suggestion
files. The DB key is also a pair that has a public key and a private key.
```

> The private DB key signs the UKI image in the Edge Installer ISO. The public DB key verifies the signature on the UKI

Reviewer: seems the first time we talk about UKI files here, maybe worth expanding?

image.

Both the public and private DB keys should be stored securely in the build pipeline of your Edge artifacts, as they are
needed during EdgeForge both during initial deployment and upgrades.

## Platform Configuration Registers (PCR) Policy Key

The PCR policy key is technically also a key pair, but you only see the private key after the key generation step. The

> public key is generated automatically during the EdgeForge process.

Reviewer: ```suggestion
public key is generated automatically during the EdgeForge process inside the generated artifacts.
```

Reviewer: Maybe we should also expand here a bit: we can talk about how the PCR policy is used.

The PCR policy is used to seal the PCR banks: as such, after installation (which seals the PCR bank with the policy) only booting EFI files with those matching measurements will be able to unseal the same banks later on following reboots.
